.gitignore

```diff
@@ -31,9 +31,9 @@ etc/prod.config.yaml
 etc/mock.config.yaml
 
 #Compiled binaries
-broker
+/broker
 build/broker
-!pkg/broker
+#!pkg/broker
```

Comment:
Should we just delete this line?

Reply:
shit um yeah. I'll do it in another PR since I just merged this one before seeing your comment.